client/client.go

```diff
@@ -40,7 +46,7 @@ func (c *Client) BroadcastTxSync(ctx context.Context, tx *sqproto.SignedTransact
 	return c.client.BroadcastTxSync(ctx, bytes)
 }
 
-func (c *Client) GetBalance(ctx context.Context, addr [20]byte) (*big.Int, error) {
+func (c *Client) GetBalance(ctx context.Context, addr [20]byte) ([]*BalanceResponse, error) {
```

Comment:
What's the reason for returning a list now? Should we rename to `GetBalances`?

Reply:
The sequencer api changed to return a list. I think because we changed to allow paying in different assets.

I think a rename to `GetBalances` probably makes sense. A `GetBalance` that specifies a specific asset might also make sense in the future and renaming will leave that option open.